app/submitter/submitter.py

```diff
@@ -31,11 +34,23 @@ def __init__(self, bucket_name: str) -> None:
         client = storage.Client()
         self.bucket = client.get_bucket(bucket_name)
 
-    def send_message(self, message: str, tx_id: str, case_id: str) -> bool:
+    def send_message(
+        self,
+        message: str,
+        tx_id: str,
+        case_id: str,
+        receipting_keys: Optional[tuple] = None,
+    ) -> bool:
         logger.info("sending message")
 
         blob = self.bucket.blob(tx_id)
-        blob.metadata = {"tx_id": tx_id, "case_id": case_id}
+        metadata = {"tx_id": tx_id, "case_id": case_id}
+
+        if receipting_keys:
+            for item in receipting_keys:
+                metadata[item] = item
```

Comment:
This is just setting the key and value to the same value... The value should be the value from metadata.

Reply:
Updated this now 👍